### ISSUE TITLE
Extraordinary Items Table Issues

### ISSUE THREAD

**Issue by JVanGo** [openlegend/core-rules#411]
So I revised the extraordinary items table and there are some mistakes. As Jonathan already mentioned, **Deadly** is systematically calculated with +2WL, while the table notes it as a +1WL. I would suggest keeping Deadly at a +2, but that's just my opinion.

Another problem in evaluating the costs was the **Special** propriety, as it is a somewhat subjective modifier, so while there may be items where I don't agree with the cost because of their special propriety, those are not listed here.

So here are the extraordinary items with WL calculation mistakes:

- **Digital Camouflage**: Invisibility 5 (WL2); Reliable (WL2) = WL4, but is listed as WL3
- **Frostblight**: Has listed **Aura 2**, but the Aura boon starts at PL4, which might have an effect on the item's WL
- **Helm of Truth**: Bolster 3 (WL1); Persistent (WL3); Cursed Truthfulness 5 (WL2) = WL6, but is listed as WL5
- **Jet Pack**: Flight 6 (WL3), but is listed as WL4
- **Soul Siphon**: Fatigued 5 (WL2), but is listed as WL3
- **Worker Bot**: Might 3 (WL2), Autonomous (WL2) = WL4, but is listed as WL5. I think though that the **Sentient** propriety was intended to be used here, which would bring it to WL5.

Lastly, I'd like to point out the **Plasmablade**, which is calculated correctly at the moment, if you don't take the Special ability into account. But I feel the Plasmablade's special ability is quite powerful and not all that hard to trigger, so I think it should be accounted for in the total WL.

**Comment by istabosz** [comment]
According to the table, deadly is +1 WL per deadly value. Are you proposing it should be +2 per level or a flat +2?

**Comment by JVanGo** [comment]
Deadly used to be +2 per amount of deadly and all the weapons are still calculated under that assumption. So either every Deadly weapon in the table needs to be recalculated or the Deadly cost needs to be raised.

I'd like to see Deadly going back to the previous cost, else deadly is an awfully cheap propriety for the amount of power it offers.

**Comment by brianfeister** [comment]
I was trying to think about where we got this mixed up @istabosz and @JVanGo and I think I figured it out just now: when we decided to make Deadly +1 WL per rank, we did so with a previous idea that Extraordinary items that were weapons would have a base value equal to their weapon WL and then the Deadly property would bump it higher, however we abandoned that and decided to treat the weapon and the Extraordinary properties as two separate items each with their own cost, rather than cumulatively increasing WL.

**Comment by brianfeister** [comment]
Alright, so final decision on the **Deadly** property. It needs to stay at +1 WL. The reason we used to have it at +2 WL was because the standard "weapons" table didn't require GM approval to cheat and get a powerful item. The deadly property on Extraordinary items requires GM approval. It's also set at a max of +3 (I'm adding the same for powerful). The problem with making it +2 is that you can get a `Creation 3` "wand of healing + many other things" for WL 2. It doesn't make sense for a **Deadly 2** (WL 4) item to cost the same as **Bracers of Movement 7** (also WL 4)

**Comment by brianfeister** [comment]
I think the real issue here for @JVanGo, @BuildAndDie and those who feel it should be +2 WL is that you just need to implement that as a homebrew in your game. And trust me, I'm not one to gift powerful items to characters. In fact, one of the reasons Open Legend didn't have magic items for so long was actually just because I typically don't give out magic items at all in games that I GM.

**Comment by brianfeister** [comment]
Ok, scratch that. Changed my mind. The problem again is the non-cumulative cost with base weapon cost. Obviously you can't have a **Deadly 1** weapon that is WL 1 (which it would be). So the new rule is 

**Deadly** WL modifer = _deadly value + 2_

This supports the basic principal that a "masterwork" or "+1" weapon is just out of reach for a 1st level character at WL 2.

**Comment by brianfeister** [comment]
@JVanGo 

- **Digital Camouflage**: Invisibility 5 (WL2); Reliable (WL2) = WL4, but is listed as WL3
> fixed

- **Frostblight**: Has listed **Aura 2**, but the Aura boon starts at PL4, which might have an effect on the item's WL
> The fact that this was wrong (needed another attribute to invoke Aura, which it didn't have) led me down a rabbit hole. The fact that the Plasmablade had to have "Disarmed 3" as a bane to being to use the Baneful property makes no sense. New rules text for **Baneful**:

_**Baneful (bane)** - When making a damaging attack with this item, you may automatically inflict a listed bane if your attack roll exceeds the target's defense by 5 or more. The bane can be triggered this way in lieu of other banes, even if the item or wielder cannot access the bane. The invoking attribute for this bane is equal to the attacking attribute._

- **Helm of Truth**: Bolster 3 (WL1); Persistent (WL3); Cursed Truthfulness 5 (WL2) = WL6, but is listed as WL5
> This item does not make sense at WL 6, the WL 5 is the kind of thing you would pay for to use against an enemy, but the Bolster **helps** your enemy. Therefore, if anything the Bolster should reduce it by 1, down to 4. But for simplicity I'll leave it at WL 5. 

_@istabosz, us breaking the math rules is probably going to agitate people, we should really consider replacing any item that has this kind of odd "half good / half bad" scenario where the WL just doesn't make sense. We can do that below where it's explained in the example items we build, but since there's no space to explain ourselves in the "basic" extraordinary items, we probably should just fix any of them that don't fall into line with normal WL calculations_

- **Jet Pack**: Flight 6 (WL3), but is listed as WL4
> Flight 6 is actually WL 3

- **Soul Siphon**: Fatigued 5 (WL2), but is listed as WL3
> WL 2 seems a bit low here, but I've fixed to adhere to the formula

- **Worker Bot**: Might 3 (WL2), Autonomous (WL2) = WL4, but is listed as WL5. I think though that the **Sentient** propriety was intended to be used here, which would bring it to WL5.
> this is WL 4 here on Github, must be behind on last update to the website (which I am)

**Comment by brianfeister** [comment]
Re-opening because @istabosz, me, and others need to come to an agreement about breaking the WL rules.

**Comment by istabosz** [comment]
1) I'm okay with only including items that use a normal WL calculation.

2) Step 4 of Extraordinary Item creation is already a sort of common sense rule, so I don't think we need to add a new one.

3) Changing the baneful property should also be reflected in the cost. Otherwise Baneful (slowed) and Baneful (death) are the same cost. I suggest the WL cost should be "1 greater than the WL cost for the bane according to the table in step 1"

**Comment by brianfeister** [comment]
@istabosz 

1. Sounds good
1. Ah, right. Good call! So do you feel good about my "Step 5" added here? https://github.com/openlegend/core-rules/blob/master/core/src/05-wealth-equipment/01-chapter-five.md#step-5-check-with-your-gm ... seems in this case that it makes things symmetrical, which is probably **great**! Though I have a feeling I didn't do a great job with the wording.

**Comment by istabosz** [comment]
I think it might be too wordy. I don't know if we have to do so much to justify the rule. I think common sense will prevail. What if we just make it congruent with step 4 of extraordinary items, like so:

Calculate the total wealth level accrued in the previous steps. Then, compare the weapon to others of similar power. If necessary, adjust the wealth level up or down slightly so that it is an accurate representation of the item’s influence on the game compared to other weapons.

**Comment by istabosz** [comment]
Actually, I've got to also take into account the current text for step 4. So, here's my full proposal:

Step 4: Calculate Wealth Level

All weapons (with the exception of unarmed strike), begin with a base wealth level of 1. From there, calculate the wealth level of the weapon based on the decisions you made in steps 2 and 3. Then, compare the weapon to others of similar power. If necessary, adjust the wealth level up or down slightly so that it is an accurate representation of the item’s influence on the game compared to other weapons.

**Comment by brianfeister** [comment]
@istabosz I was hoping to be more concise which was why I called attention to it, but

> ... adjust the wealth level up or down slightly so that it is an accurate representation of the item’s influence on the game compared to other weapons

doesn't work for me. "influence on the game" could be mechanical or story-based. We're explicitly declaring that story should take precedence, so that a Sniper Rifle isn't a common item purchased by homeless people. Perhaps:

> ... adjust the wealth level up or down slightly so that it is an accurate representation of the item’s **overall influence on the story and mechanics of your game world** compared to other **items**

**Comment by brianfeister** [comment]
TLDR; I think we should alert people to the fact that it's a combined / aggregate measurement of impact across two different scopes

**Comment by istabosz** [comment]
Gotcha. I like your edit above, but not a fan of the world mechanics. "Mechanics of your game world" doesn't seem to make sense.

What about "overall influence on the story and game play compared to other items"

**Comment by brianfeister** [comment]
Game play doesn't feel "crunchy" enough, how about

> overall influence on the story and thematically appropriate power level compared to other items

**Comment by istabosz** [comment]
That works for me. I was actually originally drafting similar wording but trying to avoid using "power level". But I don't actually think it's a big deal. Slight grammar fix:

"overall influence on the story and intended power level compared to other items"
